android/src/main/java/com/reactnativestripesdk/CardFormView.kt

```diff
@@ -46,27 +45,28 @@ class CardFormView(private val context: ThemedReactContext) : FrameLayout(contex
     cardFormView.postalCodeContainer.visibility = visibility
   }
 
-  fun setPlaceHolders(value: ReadableMap) {
-    val cardFormView = StripeCardFormViewBinding.bind(cardForm)
-
-    val numberPlaceholder = getValOr(value, "number", null)
-    val expirationPlaceholder = getValOr(value, "expiration", null)
-    val cvcPlaceholder = getValOr(value, "cvc", null)
-    val postalCodePlaceholder = getValOr(value, "postalCode", null)
-
-    numberPlaceholder?.let {
-//      multilineWidgetBinding.tlCardNumber.hint = it
-    }
-    expirationPlaceholder?.let {
-      multilineWidgetBinding.tlExpiry.hint = it
-    }
-    cvcPlaceholder?.let {
-      multilineWidgetBinding.tlCvc.hint = it
-    }
-    postalCodePlaceholder?.let {
-      cardFormView.postalCodeContainer.hint = it
-    }
-  }
+  // TODO: uncomment when ios-sdk allows for this
```

Comment:
Will this be a change of behavior that should be documented in the changelog?

Reply:
Nope, the ReactMethod that calls into this was already commented out so it's basically dead code: https://github.com/stripe/stripe-react-native/pull/832/files#diff-30505d89bcbbe719ef37810e46ecf58b541642866e602a43f39f36804e0a3ee3R41-R44

Reply:
ok, great